packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1894,7 +1918,7 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   addMedia(options = {}) {
-    if (MeetingUtil.isUnjoined(this.locusInfo) && !this.wirelessShare) {
+    if (MeetingUtil.isGuestUnjoined(this.locusInfo) && !this.wirelessShare) {
```

Comment:
Regular users will be able to add media to an unjoined meeting?

Reply:
Ya a regular user either in 1:1 or space meeting can add MEdia in IDLE state but guest are different